### PR TITLE
Move full list of extra_paired_delimiters characters to a better location

### PR DESCRIPTION
**Pull request by leonerd**
feature.pm is probably a bad place for this. It seems more related to the quote-like docs in perlop.pod

Additionally, putting it at the end of the file means most users won't have to scroll past it most of the time.

### CODE REVIEW

**Review by ilmari — APPROVED** [review]

File: pod/perlop.pod [hunk 1]
```diff
@@ -1591,6 +1591,10 @@ Note, however, that this does not always work for quoting Perl code:
 is a syntax error.  The C<L<Text::Balanced>> module (standard as of v5.8,
 and from CPAN before then) is able to do this properly.
 
+If the C<extra_paired_delimiters> feature is enabled, then Perl will
+additionally recognise a variety of Unicode characters as being paired. For
+a full list, see the L</APPENDIX> at the end of this document.
```
Comment (haarg) [inline pod/perlop.pod]: This should link directly to the `List of Extra Paired Delimiters` section.

File: regen/feature.pl [hunk 2]
```diff
@@ -956,396 +956,6 @@ =head2 The 'extra_paired_delimiters' feature
 compatibility with code that disables it). This feature is now considered
 stable.
 
-The complete list of accepted paired delimiters as of Unicode 14.0 is:
```
Comment (haarg) [inline regen/feature.pl:959]: There should be a link here to the new location of the list.

**Review by haarg — CHANGES_REQUESTED** [review]

File: pod/perlop.pod [hunk 3]
```diff
@@ -3831,4 +3835,398 @@ Here is a short, but incomplete summary:
...
+ ＜ ＞   U+FF1C, U+FF1E   FULLWIDTH LESS-THAN/GREATER-THAN SIGN
+ ［ ］   U+FF3B, U+FF3D   FULLWIDTH LEFT/RIGHT SQUARE BRACKET
+ ｛ ｝   U+FF5B, U+FF5D   FULLWIDTH LEFT/RIGHT CURLY BRACKET
+ ｟ ｠   U+FF5F, U+FF60   FULLWIDTH LEFT/RIGHT WHITE PARENTHESIS
+ ｢  ｣    U+FF62, U+FF63   HALFWIDTH LEFT/RIGHT CORNER BRACKET
+ ￫  ￩    U+FFEB, U+FFE9   HALFWIDTH RIGHT/LEFTWARDS ARROW
+ 𝄃  𝄂    U+1D103, U+1D102 MUSICAL SYMBOL REVERSE FINAL BARLINE,  MUSICAL
+                          SYMBOL FINAL BARLINE
+ 𝄆  𝄇    U+1D106, U+1D107 MUSICAL SYMBOL LEFT/RIGHT REPEAT SIGN
+ 👉 👈   U+1F449, U+1F448 WHITE RIGHT/LEFT POINTING BACKHAND INDEX
```
Comment (rwp0) [inline pod/perlop.pod:4176]: this is the best one 🙂

**Comment by leonerd** [comment]
Pod links added, and `=encoding` line fixed.

**Review by haarg — APPROVED** [review]

**Pull request merged**